### PR TITLE
feat(UI): ModifiedOn & MOdifiedBy fields for Project/COmponent/Release

### PR DESCRIPTION
**Pull request by akapti**
[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
Added `ModifiedBy` and `ModifiedOn` fields for `Project`, `Component` and `Release.

Issue: #1479 

### How To Test?

- Modify / Update the project , component and release from UI and REST API. You should be able to see the `ModifiedOn` and `ModifiedBy` fields values in UI.
- Merge / Split the `Component`, Only the component that got merged should have appropriate `ModifiedOn` and `ModifiedBy` fields values, and not the linked releases.
- These 2 fields are not exposed via `REST` API. (in order to avoid changes in response structure) 

### Checklist
Must:
- [x] All related issues are referenced in commit messages and in PR

Signed-off-by: Abdul Kapti <abdul.kapti@siemens-healthineers.com>

### CODE REVIEW

**Comment by kumarnikesh** [comment]
Test is successful.

File: frontend/sw360-portlet/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -810,6 +810,7 @@ moderator=Moderator
 moderators=Moderators
 modified=Modified
 modified.by=Modified By
+modified.date.yyyy.mm.dd=Modified date YYYY-MM-DD
```
Comment (smrutis1) [inline frontend/sw360-portlet/src/main/resources/content/Language.properties:813]: Please update it in other Language properties file as well.

**Review by smrutis1 — CHANGES_REQUESTED** [review]
One minor changes requested w.r.t properties file

File: frontend/sw360-portlet/src/main/resources/content/Language.properties (hunk 1 above)
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language.properties:813]: FYI ;  Japanese
modified.date.yyyy.mm.dd=修正日 YYYY-MM-DD

Comment (akapti) [reply frontend/sw360-portlet/src/main/resources/content/Language.properties:813]: > Please update it in other Language properties file as well.

Done. Thanks.

**Review by smrutis1 — APPROVED** [review]
Code looks good.

**Pull request merged**